common-money.lic

```diff
@@ -28,25 +28,33 @@ module DRCM
     amount
   end
 
-  def convert_currency(amount, from, to, fee = 0.05)
+  def convert_currency(amount, from, to, fee)
```

Comment:
Do you plan to put the actual convert mechanics (not just calculations) in `common-money`?

Reply:
That could be useful

Reply:
I guess the mechanics used in pay-debt are a test to see if thats a good/working approach: deposit all the money on hand then withdraw only what you need to make the exchange.